File: sort/selectionSort.py
def selection_sort(arr):
    for i in range(0, len(arr)):
        min = i
        for j in range(i+1, len(arr)):
            if arr[min] > arr[j]:
                min = j
        arr[i], arr[min] = arr[min], arr[i]
    return arr


print(selection_sort([5, 1, 7, 3, 9, 8, 4]))
# O(n^2)

File: sort/selectionSort.js
function selectionSort(arr) {
  for (i = 0; i < arr.length; i++) {
    let min = i;
    for (j = i + 1; j < arr.length; j++) {
      if (arr[min] > arr[j]) {
        min = j;
      }
    }

    let temp = arr[i];
    arr[i] = arr[min];
    arr[min] = temp;
  }
  return arr;
}

console.log(selectionSort([5, 1, 7, 3, 9, 8, 4]));
// O(n^2)
